helper/hex/hex.go

```diff
@@ -52,6 +52,31 @@ func EncodeUint64(i uint64) string {
 	return string(strconv.AppendUint(enc, i, 16))
 }
 
+// DecodeUint64 decodes a hex string with 0x prefix to uint64
+func DecodeUint64(hexStr string) (uint64, error) {
+	// remove 0x suffix if found in the input string
+	cleaned := strings.TrimPrefix(hexStr, "0x")
+
+	// base 16 for hexadecimal
+	result, err := strconv.ParseUint(cleaned, 16, 64)
+	if err != nil {
+		return 0, err
+	}
+
+	return result, nil
+}
+
+// MustDecodeUint64 decodes a hex string with 0x prefix to uint64
+// Returns panic if there is an error
+func MustDecodeUint64(hexStr string) uint64 {
```

Comment:
Please avoid adding methods that contain explicit panics

Reply:
I've added this method following the standard from this package. As you can see other decode methods (https://github.com/0xPolygon/polygon-edge/blob/develop/helper/hex/hex.go#L38) have MustDecode versions used mainly for tests. Agree we can remove this only because we don't want to risk that someone will use this i code and trigger panic.
fixed: a50aef5ac33eb44c76d87c8367a20a1158c146fe